WalletWasabi/Nito/Collections/Deque.cs

```diff
@@ -130,7 +130,7 @@ public int Capacity
 
 		set
 		{
-			if (value is null && default(T) is not null)
```

Comment:
This class got many warning, starting with the namespace.
Could you please fix the trivial ones in this PR or in another?